### DISCUSSION TITLE
New setup (no Docker)

### DISCUSSION THREAD

**Discussion by laurencijs** [Kovah/LinkAce#855] [General]
Hello everyone!

I’m absolutely new to LinkAce, and I’m starting from scratch. In my setup, Docker is not an option, so I’m sticking with a non-Docker setup. My web server is nginx. Server Ubuntu 24.04, php 8.3
[linkace.conf.txt](https://github.com/user-attachments/files/17394773/linkace.conf.txt)

I’ve been following the instructions from here: [LinkAce Setup without Docker](https://www.linkace.org/docs/v1/setup/setup-without-docker/). Everything goes fine until I try to open the web page, where I get a 500 error.

From the logs, I can see the following error:
`2024/10/16 10:38:49 [error] 1726878#1726878: *1614 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: <ip>, server: <fqdn>, request: "GET /public/setup/start HTTP/1.1", host: "<fqdn>"
`
Any idea how I can resolve this? I don’t see any setup directory with "start" under /public, nor in the zip file or the Git repo.

Thanks!

Best regards,
Lauris

**Comment by Kovah** [comment]
Hi, thanks for taking the time to try LinkAce write about the issues you have with the setup.
I did not test the non-Docker setup a lot because I just don't use it.

Have you pointed your web server to the public directory? If possible, share your configuration and censor sensitive parts.

**Reply by laurencijs** [reply]
Hi Kovah,

Thank you for your response!

I'm not sure what more to add beyond my previous post. I've attached the nginx configuration file [linkace.conf.txt](https://github.com/user-attachments/files/17394773/linkace.conf.txt), (had to rename the file as github wouldn't allow .conf) which contains everything related to the specific website. The PHP version in use is 8.3.

I am indeed trying to open /public, but there doesn't seem to be a setup directory under the public folder in the repository. Nginx seems to loop, as per the recommended config, and ultimately returns a 500 error.

If there's anything more specific you'd like me to provide, please let me know.

Thanks! Best regards,
Lauris

**Reply by Kovah** [reply]
Indeed your config is wrong.

Replace `root /var/www/linkace;` with `root /var/www/linkace/public;`.
As written in the docs, the web server must point to the `public` directory, not the linkace one below.
